src/components/Footer/Footer.js

```diff
@@ -42,62 +102,109 @@ const Footer = ({
   t,
   dispatchSetLanguage,
   dispatchSetProjection,
-}) => (
-  <RSFooter className="wkp-footer">
-    <div className="wkp-footer-left">
-      <Copyright
-        layerService={layerService}
-        format={f => `${t('Geodaten')} ${f}`}
-      />
-    </div>
-
-    <div className="wkp-footer-right">
-      <MousePosition
-        coordinatePosition="left"
-        map={map}
-        onChange={(evt, proj) => {
-          dispatchSetProjection(proj);
-        }}
-        projections={[
-          {
-            label: 'CH1093 / LV03',
-            value: 'EPSG:21781',
-            format: c => `${t('Koordinaten')}: ${numberFormat(c)}`,
-          },
-          {
-            label: 'CH1093+ / LV95',
-            value: 'EPSG:2056',
-            format: c => `${t('Koordinaten')}: ${numberFormat(c)}`,
-          },
-          {
-            label: 'Web Mercator',
-            value: 'EPSG:3857',
-            format: c => `${t('Koordinaten')}: ${numberFormat(c)}`,
-          },
-          {
-            label: 'WGS 84',
-            value: 'EPSG:4326',
-            format: c => `${t('Koordinaten')}: ${c}`,
-          },
-        ]}
-      />
-      <ScaleLine map={map} />
-      <Select
-        className="wkp-language-select"
-        options={[
-          { label: 'DE', value: 'de' },
-          { label: 'FR', value: 'fr' },
-          { label: 'IT', value: 'it' },
-          { label: 'EN', value: 'en' },
-        ]}
-        value={language}
-        onChange={(e, opt) => {
-          dispatchSetLanguage(opt.value);
-        }}
-      />
-    </div>
-  </RSFooter>
-);
+}) => {
+  const [isOpen, setIsOpen] = useState(null);
+
+  // const getDialogContent = value => {
+  //   if (value === 'Kontakt') {
+  //     return t('contact_content');
+  //   }
+  //   if (value === 'Impressum') {
+  //     return t('impressum_content');
+  //   }
+  //   if (value === 'Rechtliches') {
+  //     return t('legal_content');
+  //   }
+  //   return null;
+  // };
+
+  return (
+    <RSFooter className="wkp-footer">
+      <div className="wkp-footer-left">
+        <Copyright
+          layerService={layerService}
+          format={f => `${t('Geodaten')} ${f}`}
+        />
+
+        <div className="wkp-impressum">
+          <ActionLink onClick={() => setIsOpen('Kontakt')}>
+            {t('Kontakt')}
+          </ActionLink>
+          <ActionLink onClick={() => setIsOpen('Impressum')}>
+            {t('Impressum')}
+          </ActionLink>
+          <ActionLink onClick={() => setIsOpen('Rechtliches')}>
+            {t('Rechtliches')}
+          </ActionLink>
+
+          <Dialog
+            title={
+              <div>
+                <FaInfo /> {t(isOpen)}
+              </div>
+            }
+            classNameChildren="tm-dialog-content"
+            onClose={() => setIsOpen(null)}
+            cancelDraggable=".tm-dialog-content"
+            isOpen={isOpen}
+            isModal
+          >
+            <LegalLines
```

Comment:
I wouldn't call this component 'LegalLines' but rather some generic name, such as 'FooterDialogContent'

Reply:
LegalLines was generic. FooterDialogContent is not.  Pls go back to the old name.